<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scs</class>
 <widget class="QWidget" name="Scs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCS</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area I:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area II:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area III:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume
  (m3):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scs">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>650</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SCS</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CN:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area IV:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area V:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area VI:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area VIII:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area VII:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Area IX:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="areaTotal">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cn10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Precipitacao:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="precipitacao">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnMedio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>S:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sTotal">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="eTotal">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>520</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e10">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v10">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>520</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="v2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vTotal_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>    Chuva
Excedente
    (mm):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="iaTotal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>610</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>580</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ia:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ia3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
